lib/azure/resource_management/ARM_deployment_template.rb

```diff
@@ -33,6 +33,11 @@ def ohai_hints(hint_names, resource_ids)
         end
       end
 
+      if @platform == 'Windows'
+        hints_json['public_winrm_port'] = 5985

```

Comment:
Hi @btm,
During server creation, we can't predict if user will use 5985 (unencrypted winrm port) or 5986 (https winrm port).
So removing support for passing winrm port detail to ohai.
In `knife winrm` command help section for `winrm-port` option, it is mentioned that the default value is '5985' for 'plaintext' and '5986' for 'ssl' winrm transport.I think for now, we can let `knife winrm` command handle which port to pass.